marshall/package.json

```diff
@@ -6,6 +6,7 @@
   "dependancies": {
```

Comment:
Something with this property name is causing your dependencies to not be installed.